static/app/routes.tsx

```diff
@@ -1167,6 +1167,15 @@ function routes() {
             component={errorHandler(LazyLoad)}
           />
 
+          <Route
```

Comment:
I couldn't nest this under the dashboards/ route because we now render the DashboardDetail component in `dashboardsV2/index` where we were previously rendering `{children}`. I'm still wrapping my head around how this works

Reply:
if this new page will replace the current main dashboard page, I think we will have to change the structure a little